test-utils/src/main/java/org/neo4j/test/rule/DbmsRule.java

```diff
@@ -153,6 +154,19 @@ protected void after() {
 
     private void create() {
         databaseBuilder = newFactory();
+
+        // Allow experimental versions of Cypher and set Cypher Default Version
+        globalConfig.put(GraphDatabaseInternalSettings.enable_experimental_cypher_versions, true);
+
+        // A test may set this for the entire file itself, so we shouldn't override that
+        if (!globalConfig.containsKey(GraphDatabaseInternalSettings.default_cypher_version)) {
+            String cypherVersionEnv = System.getenv()
+                    .getOrDefault("CYPHER_VERSION", GraphDatabaseInternalSettings.CypherVersion.Cypher25.name());
```

Comment:
Should we leave this as 25 going forward?

The plan is to ask for the TC builds to use the env to test both Cypher 5 and 25 in PR builds :)

Reply:
I vote for using 5 as default for now and setting it to 25 when mono repo has flipped.